lib/vagrant-service-manager/service.rb

```diff
@@ -16,8 +16,13 @@ def initialize(app, env)
 
       def call(env)
         @app.call(env)
-        return false unless SUPPORTED_BOXES.include? @machine.guest.capability(:os_variant)
-        @service_hooks.each(&:execute)
+
+        if SUPPORTED_BOXES.include? @machine.guest.capability(:os_variant)
+          return false unless SUPPORTED_BOXES.include? @machine.guest.capability(:os_variant)
+          @service_hooks.each(&:execute)
+        end
+      rescue Vagrant::Errors::GuestCapabilityNotFound
+        # Do nothing if supported box variant not found
```

Comment:
@navidshaikh Probably, we don't want to do that who is using different box and have nothing to do with adb/cdk box. It would be unnecessary thing for the user.
And, if we do that user might come back complaining that vsm is throwing this error etc.

We might add debug logs here which might be visible during `--debug` option only but again is it necessary since its not relevant for us if box is not adb/cdk altogether.